ta/pkcs11/src/processing_asymm.c

```diff
@@ -453,13 +508,47 @@ enum pkcs11_rc step_asymm_operation(struct pkcs11_session *session,
 			goto out;
 		}
 		break;
+
+	case PKCS11_CKM_RSA_PKCS:
+	case PKCS11_CKM_MD5_RSA_PKCS:
+	case PKCS11_CKM_SHA1_RSA_PKCS:
+	case PKCS11_CKM_SHA224_RSA_PKCS:
+	case PKCS11_CKM_SHA256_RSA_PKCS:
+	case PKCS11_CKM_SHA384_RSA_PKCS:
+	case PKCS11_CKM_SHA512_RSA_PKCS:
+		/*
+		 * Constraints on key types and the length of the data for
+		 * these mechanisms are summarized in the following table.
```

Comment:
remove reference to a table, there is no table...

Reply:
OK

Reply:
Actually removed the whole comment as I though it is more confusing to be there.

If one goes to look at the table in specification it is much clearer that just check the input for key size for verify. Output is handled differently elsewhere.